docs/quickstart.md

```diff
@@ -60,9 +60,8 @@ import TabItem from '@theme/TabItem';
 
 ```bash
 $ iasql new
-
 ✔ IaSQL db name · startup
-✔ Pick AWS region · us-east-1
+✔ Pick AWS region · us-west-2
```

Comment:
the public ecr repository has to be in `us-east-1` so let's leave that for everything?